concordium-node/src/stats_export_service.rs

```diff
@@ -93,6 +93,25 @@ pub struct StatsExportService {
     /// - `"duplicate"` duplicate consensus messages (this is not accounting
     ///   deduplication of the network layer).
     pub received_messages: IntCounterVec,
+    /// Total number of consensus messages sent. Labelled with message type
+    /// (`message=<type>`).
+    ///
+    /// Possible values of `message` are:
+    /// - `"block"`
+    /// - `"transaction"`
+    /// - `"finalization record"`
+    /// - `"finalization message"`
+    /// - `"catch-up status message"`
+    pub sent_messages: IntCounterVec,
+    /// Current number of soft banned peers.
+    pub soft_banned_peers: IntGauge,
+    /// Total number of peers connected since startup.
+    pub peers: IntCounter,
+    /// General node information, so far only contains a label `version` with
+    /// the version of the node.
```

Comment:
I'm not very keen on this "general" description. If we don't have any specifics that would fit this, then please document what it is, and not "it could be more" with no specifics.

Reply:
This is also missing the description of how this is used.

There is only one value for this, 1, and the label is the actual data.

I'm not really a fan of this use of Prometheus. It is clearly hacky to have a boolean flag for each possible version of the node.

Reply:
I'll make it more specific.

I understand why you dislike this approach, but it is from the Prometheus documentation https://prometheus.io/docs/instrumenting/writing_exporters/#target-labels-not-static-scraped-labels.
We could choose to not have this information, I guess.